plugins/kubernetes/app/controllers/kubernetes_project_controller.rb

```diff
@@ -1,16 +1,15 @@
 class KubernetesProjectController < ApplicationController
-  before_action :authorize_deployer!
-  before_action :project
+  include ProjectLevelAuthorization
+  before_action :authorize_project_deployer!
```

Comment:
That's a good point. We could hide all tabs except the Dashboard, or show all the tabs but hide the buttons for creating new Roles and Releases.

Food for thoughts... I think we can squeeze that in easily, anyway, because all this files are ERBs.